lang/rust/patches/0002-rustc-bootstrap-cache.patch

```diff
@@ -38,15 +41,3 @@
          let cache_dir = cache_dst.join(key);
          if !cache_dir.exists() {
              t!(fs::create_dir_all(&cache_dir));
-@@ -671,7 +680,10 @@ download-rustc = false
-         let llvm_assertions = self.llvm_assertions;
- 
-         let cache_prefix = format!("llvm-{llvm_sha}-{llvm_assertions}");
--        let cache_dst = self.out.join("cache");
-+        let cache_dst = match env::var_os("OPENWRT_RUSTC_BOOTSTRAP_CACHE") {
```

Comment:
I wonder why this part was removed.

Reply:
Ouch, it's a bug I did when was manually rebasing the patches. Shame on me!
Should I prepare the fix?

Reply:
Would be great if you can create a PR for it ;)

Reply:
@1715173329 sorry for the hussle, PR is ready - #23581

Reply:
No worries! Thank you!